scripts/check-license.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+ROOT_PATH=$PWD
+LICENSE_CHECKER_PATH=$PWD/scripts/license-checker
+
+cd $LICENSE_CHECKER_PATH
+npm ci
+./license-checker generate $ROOT_PATH > $ROOT_PATH/license-lock && git diff --exit-code
+./license-checker license "Mapbox Base" $ROOT_PATH > $ROOT_PATH/LICENSE.thirdparty && git diff --exit-code
```

Comment:
Would it make sense to provide artefacts with changes to apply? :thinking:

Reply:
@mr1sunshine not sure how to set up artifacts on Travis CI. Also, running `scripts/check-license.sh` should be enough to generate the same expectation.